Mathlib/LinearAlgebra/AffineSpace/Basis.lean

```diff
@@ -275,6 +273,24 @@ instance instAddAction : AddAction V (AffineBasis ι k P) :=
   congr! 1
   rw [vadd_vsub_assoc, neg_add_eq_sub, vsub_vadd_eq_vsub_sub]
 
+section SMul
+variable [Group G] [SMul G k] [DistribMulAction G V] [SMulCommClass G k V] (b : AffineBasis ι k P)
+  {s : Finset ι} {i j : ι} (e : ι ≃ ι')
+
+instance instSMul : SMul G (AffineBasis ι k V) where
+  smul a b :=
+    { toFun := a • ⇑b,
+      ind' := b.ind'.smul,
+      tot' := by
+        rw [Pi.smul_def, ← smul_set_range, ← AffineSubspace.smul_span, b.tot,
+          AffineSubspace.smul_top (Group.isUnit a)] }
+
+@[simp, norm_cast] lemma coe_smul (a : G) (b : AffineBasis ι k V) : ⇑(a • b) = a • ⇑b := rfl
```

Comment:
Now that we have #14945, let's add a lemma about `basisOf` too, and perhaps `coord` or whatever else we did for `vadd`.